### ISSUE TITLE
[ListView] Keyboard navigation focus not working in Safari

### ISSUE THREAD

**Issue by nantunes** [lumada-design/hv-uikit-react#1766]
**Describe the bug**
In macOS Safari, when navigating the Asset Inventory's List View using the keyboard, the focus outline around the list row isn't visible and some of the inner components focus outline are cut-out. Also, when using the up and down keys, no focus outline is visible.

**Screenshots**
![out](https://user-images.githubusercontent.com/209365/87058617-30a30500-c200-11ea-9416-aeef3d0ae20b.gif)
(the animation shows it working in Chrome and not working in Safari)

**Enviroment (please complete the following information):**
 - Device: Laptop
 - OS: macOS Catalina
 - Browser: Safari 13.1.1
